libyul/AssemblyStack.cpp

```diff
@@ -274,6 +274,44 @@ std::pair<MachineAssemblyObject, MachineAssemblyObject> AssemblyStack::assembleW
 	return {std::move(creationObject), std::move(deployedObject)};
 }
 
+std::pair<std::shared_ptr<evmasm::Assembly>, std::shared_ptr<evmasm::Assembly>> AssemblyStack::assembleEVM(optional<string_view> _deployName) const
+{
+	yulAssert(m_analysisSuccessful, "");
+	yulAssert(m_parserResult, "");
+	yulAssert(m_parserResult->code, "");
+	yulAssert(m_parserResult->analysisInfo, "");
+
+	evmasm::Assembly assembly;
+	EthAssemblyAdapter adapter(assembly);
+	compileEVM(adapter, m_optimiserSettings.optimizeStackAllocation);
+
+	optional<size_t> subIndex;
+
+	// Pick matching assembly if name was given
+	if (_deployName.has_value())
```

Comment:
Basically we could just have a helper in `evmasm` for `findDeployed(name)` doing this.